### PR TITLE
modifying the documentation as per Issue#1903

### PR DESCRIPTION
**Pull request by rohitkochar**

### CODE REVIEW

File: docs/content/design/segments.md [hunk 1]
```diff
@@ -43,18 +43,19 @@ group-by operations, so each dimension requires the following
 three data structures:
 
 1. A dictionary that maps values (which are always treated as strings) to integer IDs,
-2. For each distinct value in the column, a bitmap that indicates which rows contain that value, and
-3. A list of the column’s values, encoded using the dictionary in 1.
+2. A list of the column’s values, encoded using the dictionary in 1.
```
Comment (rasahner) [inline docs/content/design/segments.md]: this should end in ", and" and remove the ", and" from number 3.

File: docs/content/design/segments.md [hunk 2]
```diff
@@ -43,18 +43,19 @@ group-by operations, so each dimension requires the following
...
+
 
 Why these three data structures? The dictionary simply maps string
 values to integer ids so that the values in 2 and 3 can be
-represented compactly. The bitmaps in 2 -- also known as *inverted
+represented compactly. The bitmaps in 3 -- also known as *inverted
 indexes* allow for quick filtering operations (specifically, bitmaps
 are convenient for quickly applying AND and OR operators). Finally,
-the list of values in 3 are needed for *group by* and *TopN*
+the list of values in 2 are needed for *group by* and *TopN*
```
Comment (rasahner) [inline docs/content/design/segments.md]: this should be "the list .... is needed"

**Comment by rohitkochar** [comment]
incorporated the review comments.

**Comment by fjy** [comment]
:+1: for me. @rasahner are you cool with the changes

**Comment by rasahner** [comment]
+1

**Comment by fjy** [comment]
@rohitkochar Just realized we'll need a CLA: druid.io/community/cla.html

**Comment by rohitkochar** [comment]
CLA done.

**Pull request merged**